### DISCUSSION TITLE
What about tauri?

### DISCUSSION THREAD

**Discussion by tiziodcaio** [Heroic-Games-Launcher/HeroicGamesLauncher#2293] [Ideas]
For more infos, https://tauri.app

**Comment by imLinguin** [comment]
It's cool but no one will volunteer to port already big and complex app into `Rust`

**Reply by bogdanbiv** [reply]
Just for the record, Rust is needed only for computational intensive tasks. Tauri is smaller in file size because it does not need to carry the whole Chromium browser over, it needs just a webview instance. https://blog.logrocket.com/rust-solid-js-tauri-desktop-app/ 

That being said, it's clear porting a large app takes *some* work, it's unclear now what that would entail
